assets/cms/components/IconButton.vue

```diff
@@ -0,0 +1,68 @@
+<template functional>
+    <span class="button-container">
+        <button
+            @click='listeners.click'
+            :class="[{ 'btn btn-light' : props.format === 'floating',
+                    'btn btn-outline-secondary' : props.format === 'outline',
```

Comment:
Can you fix the spacing on this so that items are each on their own line and use the proper code style for these?

```
<button class='btn' :class="{
        'btn-light': props.format === 'floating',
        ...
    }">
```

Reply:
so do we keep 'btn' outside because is repeated...